### PR TITLE
rollout: proofing on 5 articles

### PR DESCRIPTION
**Pull request by landism**

### CODE REVIEW

File: docs/rollout/shiny-feature.md [hunk 1]
```diff
@@ -26,12 +26,11 @@ Here's a few ideas to deliver a wow feature. [Reach out to us](https://calendly.
...
+* You've had some success converting users, but many of them tell you the benefits of Tilt weren't obvious at first.
 
 **Skip this if**
+* Users are already quickly onboarding to Tilt and understanding the benefits right away.
 
-- Users are already quickly onboarding to Tilt and understanding the benefits right away.
-
-**You know it's successful when** you consistently see more aha moments from new users.
+**You know it's successful when**
+* You consistently see more aha moments from new users.
```
Comment (maiamcc) [inline docs/rollout/shiny-feature.md:36]: Idk what the correct punctuation is but it's not this. maybe `"Aha!" moments` or `"aha moments"` or `"aha!" moments`?

**Review by maiamcc — APPROVED** [review]

File: docs/rollout/prioritize-inactive.md [hunk 2]
```diff
@@ -3,8 +3,7 @@ title: Prioritize Inactive Users
 layout: rollout
 ---
 
-
-Here's a guideline for how to triage work that's surprising and counterintuitive: during roll out prioritize inactive users, i.e. people who tried Tilt but stopped using it. This can be surprising and counterintuitive; e.g. shouldn't you focus on your most active users? This doc explains how prioritizing inactive users puts your attention on the most pressing issues and can drive adoption.
+Here's a guideline for how to triage work that's surprising and counterintuitive: during roll out, prioritize inactive users, i.e. people who tried Tilt but stopped using it. This isn't often one's first instict--shouldn't you focus on your most active users? This doc explains how prioritizing inactive users puts your attention on the most pressing issues and can drive adoption.
```
[the remark this answers is not in the document]
Comment (victorwuky) [reply docs/rollout/prioritize-inactive.md:6]:
```suggestion
Here's a guideline for how to triage work that's surprising and counterintuitive: during roll out, prioritize inactive users, i.e. people who tried Tilt but stopped using it. This isn't often one's first instinct--shouldn't you focus on your most active users? This doc explains how prioritizing inactive users puts your attention on the most pressing issues and can drive adoption.
```

**Review by victorwuky — APPROVED** [review]

File: docs/rollout/adoption-skyline-charts.md [hunk 3]
```diff
@@ -16,9 +16,10 @@ We think [active hours](../rollout/active-hours) is an effective metric to repre
...
-- You are want to further insight into understanding how your users are adopting Tilt.
+* You want to back your Tilt-related decisions with actual data.
 
 **Skip this if**
-- You're not ready to track user adoption just yet.
+* You're not ready to track user adoption just yet.
 
-**You know it's successful when** you find yourself returning to the chart week over week to see any changes.
+**You know it's successful when**
+* You find yourself returning to the chart week over week to see any changes.
```
[the remark this answers is not in the document]
Comment (landism) [reply docs/rollout/adoption-skyline-charts.md:25]: Yeah I was tempted to make some change like that - "you're looking at the chart" is not really a sign of impact.

I had considered "this chart helps you track down inactive users" but then figured that turns this whole page into an implementation detail of "prioritize inactive users". Absent any other success criteria, I'm actually leaning towards just getting rid of this page and making it a section of "prioritize inactive users". That feels like a bigger move than mere proofreading, so I didn't want to tackle it as a side task. Maybe that's perfect vs good.

I've added "and you find yourself making decisions based on what you find", which seems like a reasonable middle ground for now.

File: docs/rollout/active-hours.md [hunk 4]
```diff
@@ -10,6 +10,6 @@ Choosing a good metric to track adoption is difficult. We recommend "Active Hour
 * Actionable. The data that generates Active Hours also lets you find inactive users.
 * Extensible. We can extend this metric to account for more specifics.
 
-It's easy to track Hours Active with Tilt. We've built tools to make it easy to record this with Tilt and Tilt Cloud. Contact us to get access before it's generally available.
+It's easy to track Active Hours with Tilt. We've built tools to make it easy to record this with Tilt and Tilt Cloud. [Contact us](https://calendly.com/dbentley/tilt-enterprise) to get access before it's generally available.
```
[the remark this answers is not in the document]
Comment (landism) [reply docs/rollout/active-hours.md:13]: ehhh that feels like it'd make the sentence messy and is already covered elsewhere

File: docs/rollout/active-hours.md [hunk 5]
```diff
@@ -10,6 +10,6 @@ Choosing a good metric to track adoption is difficult. We recommend "Active Hour
 * Actionable. The data that generates Active Hours also lets you find inactive users.
```
[the remark this answers is not in the document]
Comment (landism) [reply docs/rollout/active-hours.md:10]: > remove "calendar hours"

I'm guessing this should say 'remove "calendar"'

> I don't think this explanation is useful w/o the context of "why don't we just track HOURS hours"?

I actually found this whole section a bit offputting. It defines a new metric and then lists FIVE bullet points explaining why we suggest this metric, but none of them clearly distinguish this metric vs HOURS hours.

> it's just technical details

This is tricky - it is kind of just technical details, but the article is framed as "we are suggesting this metric", as if the reader might implement any metric and we are trying to convince them of this one. At that point, I think the technical details / actual definition of the metric is pretty central to the article! We should probably just reframe it as "This is what Tilt uses"

File: docs/rollout/prioritize-inactive.md [hunk 6]
```diff
@@ -22,7 +21,8 @@ The actual low-hanging fruit is Inactive Users.
 * Potential Users are likely to run into the same issues that block your current Inactive Users. Solve your existing issues before looking for new ones.
```
[the remark this answers is not in the document]
Comment (landism) [reply docs/rollout/prioritize-inactive.md:21]: > ...is this true? This seems like a weird claim

> I don't think this is right

These are more controversial than I want to dig into in a proofing PR and seem like conversations for Dan.

File: docs/rollout/prioritize-inactive.md [hunk 7]
```diff
@@ -42,4 +42,5 @@ You shouldn't totally ignore other users, even if you're prioritizing Inactive U
 **Skip this if**
 * You are confident that you understand every AppDev's use case and how to triage the impact of issues. We see this mostly when DevEx is part-time, e.g. on a team of 5-10.
```
[the remark this answers is not in the document]
Comment (landism) [reply docs/rollout/prioritize-inactive.md:43]: Yeah I have no idea what this is trying to say. I think the second sentence could just be dropped.

I feel like the first sentence is setting too high a bar with the "every" that is essentially just saying "always use this!".

I just replaced the whole thing with "You are confident your Inactive Users don't have problems worth solving"

File: docs/rollout/shiny-feature.md (hunk 1 above)
Comment (landism) [reply docs/rollout/shiny-feature.md:36]: image
🤷‍♂️

File: docs/rollout/active-hours.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (landism) [reply docs/rollout/active-hours.md]: Gonna punt this just to get this into master. Happy to address this in a future PR, but not really sure I'm the right owner for it anyway.

**Pull request merged**